default.py

```diff
@@ -350,6 +357,43 @@ def select_version(self, game_versions):
         else:
             return None
 
+    def has_inputstream_adaptive(self):
+        """Checks if InputStream Adaptive is installed and enabled."""
+        payload = {
+            'jsonrpc': '2.0',
+            'id': 1,
+            'method': 'Addons.GetAddonDetails',
+            'params': {
+                'addonid': 'inputstream.adaptive',
+                'properties': ['enabled']
+            }
+        }
+        response = xbmc.executeJSONRPC(json.dumps(payload))
+        data = json.loads(response)
+        if 'error' not in data and data['result']['addon']['enabled']:
+            addon_log('InputStream Adaptive is installed and enabled.')
+            return True
+        else:
+            addon_log('InputStream Adaptive is not installed and/or enabled.')
```

Comment:
Please add "Disabling" to the end of this log message. We should note somewhere that we're silently changing the user's addon settings.